scripts/screeps_rl_simulator_harness.py

```
@@ -5442,6 +5536,26 @@ def _run_variant(
+        )
+        token = smoke.update_token_from_headers(token, roundtrip.headers)
+        active_code_readback = private_simulator_active_code_readback_summary(
+            uploaded_code_text,
+            roundtrip.payload,
+            branch=api_branch,
+            http_status=roundtrip.status,
+        )
+        write_json_atomic(safe_run_root / "active_code_readback.json", active_code_readback)
+        active_code_error = private_simulator_active_code_readback_error(active_code_readback)
+        if active_code_error is not None:
+            raise RuntimeError(active_code_error)
```

Ruff (0.15.14)

[warning] Abstract `raise` to an inner function

(TRY301)